src/views/UserDetails.vue

```diff
@@ -9,24 +9,62 @@
                 <user-card
                   :name="user.name"
                   :email="user.email"
-                  :resourcesAndQuantities="resourceItem"
+                  :resourcesAndQuantities="resourceItemResources"
                 ></user-card>
               </suspense>
-              <resource-availability-table
-                :tableColumns="userTableColumns"
-                :resourceItem="resourceItem"
-                :user="user"
-              ></resource-availability-table>
-
-              <v-card class="elevation-12 mt-4">
-                <div class="text-center">
-                  <h1>{{ user.name }}'s products table</h1>
-                </div>
-
-                <products-table
-                  :products="userProducts"
-                />
-              </v-card>
+
+              <div class="d-flex justify-center mt-10">
+                <v-btn
+                  color="red"
+                  @click="
+                    () => (isResourceTableVisible = !isResourceTableVisible)
+                  "
+                  >{{
+                    isResourceTableVisible ? "Hide Resources" : "Show Resources"
+                  }}</v-btn
+                >
+                <v-btn
+                  color="green"
+                  @click="
+                    () => (isProductsTableVisible = !isProductsTableVisible)
+                  "
+                  >{{
+                    isProductsTableVisible ? "Hide Products" : "Show Products"
+                  }}</v-btn
+                >
+              </div>
+
+              <transition>
+                <resource-availability-table
+                  v-if="isResourceTableVisible"
+                  :tableColumns="tableColumnsResources"
+                  :resourceItem="resourceItemResources"
+                  :user="user"
+                ></resource-availability-table>
+              </transition>
+
+              <transition>
+                <v-card class="elevation-12 mt-4" v-if="isProductsTableVisible">
+                  <div class="text-center">
+                    <h1>{{ user.name }}'s products table</h1>
+                  </div>
+
+                  <products-table
+                    :products="userProducts"
+                    :additionalColumnsRight="disassemblyColumns"
+                  >
+                    <template v-slot:item.disassembly="{ item }">
+                      <v-btn
+                        variant="plain"
+                        :disabled="item.contentOf === 'Yes'"
+                        @click="disassemblyProduct(item)"
+                      >
+                        <v-icon size="25">mdi-cart-off</v-icon>
+                      </v-btn>
+                    </template>
```

Comment:
i can extract it into the Products Table but we will have more logic there anyways, shall I do it?

Reply:
but we are using <products-table> in 4 places - ProductsContentDialog, ProductsDialog, Products, UseDetails. If we place it in Products table, it will be visible. I can try to only try to extract the template.

Reply:
just pushed new commit with that change